README.md

```diff
@@ -0,0 +1,378 @@
+
+        ```
+        curl -X PUT "http://10.62.59.150:7070/iso?name=test.iso" --upload-file static/files/iso/centos-7.0.iso
+        Uploaded 10 %
+        Uploaded 20 %
+        Uploaded 30 %
+        Uploaded 40 %
+        Uploaded 50 %
+        Uploaded 60 %
+        Uploaded 70 %
+        Uploaded 80 %
+        Uploaded 90 %
+        Uploaded 100 %
+        Upload finished!
+        ```
+
+    3. Delete a iso file that is in the store. One parameter is needed.
+        * name: in query or in body, the name of the iso will be deleted.
+
+        ```
+        curl -X DELETE "http://10.62.59.150:7070/iso?name=test.iso"
+        {
+            "name": "centos-7.0.iso",
+            "size": "4.15 GB",
+            "upload": "2016-10-18T18:02:50.769Z"
+        }
+        ```
+
+* Microkernel file management
+The Microkernel file management API works similar like iso file management API.
+
+    1. Get/list install microkernel files.
+
+        ```
+        curl -X GET "http://10.62.59.150:7070/microkernel"
+        [
+          {
+            "name": "base.trusty.3.16.0-25-generic.squashfs.img",
+            "size": "61.11 MB",
+            "uploaded": "2016-11-10T16:44:10.311Z"
+          },
+          {
+            "name": "discovery.overlay.cpio.gz",
+            "size": "8.59 MB",
+            "uploaded": "2016-11-10T16:44:10.351Z"
+          },
+          {
+            "name": "initrd.img-3.16.0-25-generic",
+            "size": "23.24 MB",
+            "uploaded": "2016-11-10T16:44:10.607Z"
+          },
+          {
+            "name": "vmlinuz-3.16.0-25-generic",
+            "size": "6.34 MB",
+            "uploaded": "2016-11-10T16:44:11.939Z"
+          }
+        ]
+        ```
+
+    2. Upload an microkernel file. One parameter is needed.
+        * name: in query or in body, the name of the microkernel that will be shown in the store.
+
+        ```
+        curl -X PUT "http://10.62.59.150:7070/microkernel?name=vmlinuz-3.16.0-25-generic" --upload-file vmlinuz-3.16.0-25-generic
+        Uploaded 10 %
+        Uploaded 20 %
+        Uploaded 30 %
+        Uploaded 40 %
+        Uploaded 50 %
+        Uploaded 60 %
+        Uploaded 70 %
+        Uploaded 80 %
+        Uploaded 90 %
+        Uploaded 100 %
+        Upload finished!
+        ```
+
+    3. Delete a microkernel file that is in the store. One parameter is needed.
+        * name: in query or in body, the name of the microkernel will be deleted.
+
+        ```
+        curl -X DELETE "http://10.62.59.150:7070/microkernel?name=vmlinuz-3.16.0-25-generic"
+        {
+            "name": "vmlinuz-3.16.0-25-generic",
+            "size": "6.34 MB",
+            "uploaded": "2016-11-10T16:44:11.939Z"
+        }
+        ```
+
+### Southbound
+
+The southbound is all about static file server. It's by default listen at 0.0.0.0:9090. It also expose a GUI so that if you navigate to http://0.0.0.0:9090/ using your favorate web browser, you will get files and directories listed. 
```

Comment:
does it need to say it's using default RackHD static server? which means it's not developed in this repo?